index.yaml

```diff
@@ -1171,4 +1171,30 @@ entries:
     urls:
     - https://github.com/openshift-helm-charts/charts/releases/download/redhat-wildfly-1.4.0/redhat-wildfly-1.4.0.tgz
     version: 1.4.0
+  - annotations:
+      charts.openshift.io/digest: sha256:0fc18ecec3ca14a3fe4bcd7dd097e50f34945f412a476df34a60a235f02671fc
+      charts.openshift.io/lastCertifiedTimestamp: '2021-10-26T13:47:49.146017+00:00'
+      charts.openshift.io/provider: WildFly
```

Comment:
Should this be `Red Hat`?

Reply:
No, discussed with @jmesnil  and they want provider to show in the catalog as Wildfly, since it is a community chart, it makes sense, we need to figure out how to migrate charts from redhat/wilfly to wildfly/wildfly without affecting current users.